report/src/main/webapp/app/layout/toolbar/toolbarTop.controller.js

```diff
@@ -17,36 +17,95 @@
  */
 define([], function () {
   'use strict';
-  return ['$rootScope', 'suiteInfoService', 'metadataAccessService',
-    ToolbarTopController];
+  return ['$rootScope', '$http', 'suiteInfoService', 'metadataAccessService',
+    ToolbarTopController
+  ];
 
-  function ToolbarTopController($rootScope, suiteInfoService,
-      metadataAccessService) {
+  function ToolbarTopController($rootScope, $http, suiteInfoService,
+    metadataAccessService) {
     var vm = this;
 
     $rootScope.$on('metadata:changed', updateToolbar);
-    $('[data-toggle="popover"]').popover({
+    $("[data-toggle='popover']").popover({
       placement: 'bottom'
     });
 
+    var suiteHeaders;
+    getSuiteHistory(suiteHeaders, $rootScope, $http);
     updateToolbar();
 
     /***************************************
      ***********  Private methods  *********
      ***************************************/
 
     function updateToolbar() {
+      document.getElementsByClassName('suite-history-container')[0].style.display = 'none';
+      document.getElementsByClassName('overlay')[0].style.display = 'none';
+      vm.showSuiteHistory = function () {
+        if (isSuiteHistoryVisible()) {
+          document.getElementsByClassName('suite-history-container')[0].style.display = 'none';
+          document.getElementsByClassName('overlay')[0].style.display = 'none';
+        } else {
+          document.getElementsByClassName('suite-history-container')[0].style.display = 'block';
+          document.getElementsByClassName('overlay')[0].style.display = 'block';
+        }
+      };
       vm.suiteInfo = suiteInfoService.getInfo();
       vm.suiteStatistics = metadataAccessService.getSuite();
       if (vm.suiteStatistics.patternCorrelationId) {
         vm.pattern = {
           name: vm.suiteStatistics.patternCorrelationId,
           url: 'report.html?company=' + vm.suiteStatistics.company +
-          '&project=' + vm.suiteStatistics.project +
-          '&correlationId=' + vm.suiteStatistics.patternCorrelationId
+            '&project=' + vm.suiteStatistics.project +
+            '&correlationId=' + vm.suiteStatistics.patternCorrelationId
         };
       }
     }
 
+    function buildApiPath($allParametersList) {
+      return 'http://aet-vagrant' + '/api' + '/history' + '?' +
+        $allParametersList[0] + '&' +
+        $allParametersList[1] + '&' +
+        $allParametersList[2];
```

Comment:
The path is wrong when correlationId is used.
Example: `/report.html?company=xxx&project=xxx&correlationId=xxx`